spec/directedGraph_test.js

```diff
@@ -0,0 +1,50 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+
+import DirectedGraph from '../src/DirectedGraph'
+
+chai.use(chaiChange)
+
+describe('DirectedGraph', () => {
+  const myGraph = new DirectedGraph()
+  let vertices = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p']
+  let directions = [{vertex1:'a', vertex2:'b', weight:1},{vertex1:'a', vertex2:'c', weight:3},{vertex1:'a', vertex2:'d', weight:2},
+  {vertex1:'b', vertex2:'e', weight:2},{vertex1:'b', vertex2:'f', weight:3},
+  {vertex1:'b', vertex2:'g', weight:1},{vertex1:'b', vertex2:'l', weight:2},
+  {vertex1:'c', vertex2:'i', weight:3},{vertex1:'d', vertex2:'j', weight:2},
+  {vertex1:'d', vertex2:'d', weight:3},{vertex1:'e', vertex2:'m', weight:2},
+  {vertex1:'f', vertex2:'n', weight:3},{vertex1:'g', vertex2:'m', weight:1},
+  {vertex1:'g', vertex2:'o', weight:1},{vertex1:'h', vertex2:'m', weight:2},
+  {vertex1:'h', vertex2:'p', weight:2},{vertex1:'h', vertex2:'q', weight:1},
+  {vertex1:'i', vertex2:'r', weight:2},{vertex1:'j', vertex2:'k', weight:3}]
+
+  it('exists', () => {
+    expect(DirectedGraph).to.be.a('function')
+  })
+
+  context('addVertex() && hasVertex()', () => {
```

Comment:
pulled down your branch and this test isn't passing...can't quite figure out why. Schedule a coaching session and if we can't figure it out we can grab an SEP